ci-operator/step-registry/openstack/rotate/cloud-credentials/openstack-rotate-cloud-credentials-commands.sh

```diff
@@ -56,7 +56,4 @@ oc wait --timeout=5m --for=condition=Progressing=false $(oc get clusteroperator
 info 'Revoking the credentials that were used so far...'
 delete_application_credential "$ORIGINAL_CLOUDS_YAML"
 
-info 'Revoking the unrestricted credentials...'
-delete_application_credential "$UNRESTRICTED_CLOUDS_YAML"
```

Comment:
Why is this removed?

Reply:
We used to delete the unrestricted appcreds in the provisioning step. With this PR, we need them in the deprovisioning step in order to do the cleanup.